tests/reference/asr-structs_10-cb8a283.stderr

```diff
@@ -1,9 +1,5 @@
-semantic error: Struct member functions are not supported
- --> tests/errors/structs_10.py:7:5 - 8:24
+semantic error: The type 'StringIO' is undeclared.
```

Comment:
I think for struct, we shouldn't allow member functions.

@certik, What do you think?